src/servers/public-api/fines/fines.controller.ts

```diff
@@ -49,9 +49,16 @@ export class FinesController {
   ) {
     try {
       const { loanID } = request.params;
-      const fine = await services.fines.searchFine({
-        loanId: Number(loanID),
-      });
+      let page = Number(request.query.page);
```

Comment:
Please make use of the serializePagination function that accepts Express request context object